assets/stylesheets/babble.scss

```diff
@@ -223,6 +262,19 @@ li.babble-post .babble-post-content a.mention {
     font-size: 18px;
     right: -5px;
   }
+  &.emoji {
+    top: 2px;
+  }
+  &.upload {
+    top: 25px;
```

Comment:
The edit message stuff will gack with this. I'm wondering if we should just disable adding uploads on edit.